updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -207,6 +197,31 @@ def all_versions_ignored?(dependency, checker)
         true
       end
 
+      # if the latest update is greater than the update-types, then it should not be in the group, but
+      # be an individual PR, or another group that fits it
+      def include_in_group?(group, dependency, checker)
+        return true unless group.rules["update-types"]
```

Comment:
If there's no `rules["update-types"]` it always goes in the group? Why is that?

Reply:
Then they didn't specify a semver grouping. The prod/dev and pattern checks happen before we get to this point, so there's no need to check anything else here. 

Should I change the method name to something more semver specific for clarity? `is_semver_allowed?`

Reply:
Should we pull this check out of `include_in_group?` maybe? Feels like it's a different concern. Idk, I don't feel strongly, we can also leave a comment explaining it

Reply:
Like so?

```ruby
return [] unless group.rules["update-types"] && semver_allows_grouping?(group, dependency, checker)
```

I kind of like it in the method, I added comments and made the method name more obvious that it's applying semver  grouping rules.